frontend/13f-app/src/components/MainSection.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import '../App.css';
+import { Button } from './Button';
+import SearchBox from './SearchBox';
+import './MainSection.css';
+
+const divStyle = { float: 'left', padding: '20px', margin: '20px'}
+
+function MainSection() {
+    return (
+        <>
+            <div className='main-container'>
+                <h1>COMPANY FINANCIAL HOLDINGS</h1>
+                <p>Search 13F Filings by Company Name and Date</p>
+                <SearchBox />
+                <div style={divStyle}>
+                <input type='date' className='date'></input>
+                <input type='date' className='date'></input>
```

Comment:
Populate this with a default value, otherwise it won't be usable.

```suggestion
                const today = new Date();
                ...
                <input required type='date' name="startDate" className='date' defaultValue={today}></input>
                <input required type='date' name="endDate"  className='date' defaultValue={today}></input>
```